src/cratedb/mod.rs

```diff
@@ -0,0 +1,130 @@
+use std::{borrow::Cow, collections::HashMap};
+
+use testcontainers::{core::WaitFor, Image};
+
+const NAME: &str = "crate";
+const TAG: &str = "5.8";
+
+/// Represents a CrateDB docker instance.
+/// It is based on the official [`crate docker image`](https://hub.docker.com/_/crate)
```

Comment:
Looks a bit like a typo

```suggestion
/// It is based on the official [`crate` docker image](https://hub.docker.com/_/crate)
```